compiler-rt/test/profile/Windows/instrprof-binary-correlate.c

```diff
@@ -0,0 +1,41 @@
+// REQUIRES: target={{.*windows-msvc.*}}
```

Comment:
Do we need this? I think tests in the Windows/ dir will only run on windows hosts anyway.

Reply:
Yes, for mingw, `-dll` doesn't work. It uses `-shared`